src/ICeWall/doc/load_library.md

```diff
@@ -0,0 +1,12 @@
+## ICeWall load_library
+### Synopsys
+```
+  % ICeWall load_library <file_name>
+```
+### Description
+Loads the data required to describe the padcells that are available to be used to build a padring
```

Comment:
Sorry, I'm just having trouble interpreting this sentence 🤷‍♂️

Reply:
Reworked the description to try to make it a bit clearer